### ISSUE TITLE
Talkback is not announcing the complete information of list items in Projects screen: A11y_.NET 9 template_Hambureger-Projects Options_Talkback.

### ISSUE THREAD

**Issue by keerthiduvvuri** [dotnet/maui#30831]
### GitHub Tags:
#A11yTCS; #A11yMAS; #WCAG1.3.1; #A11ySev2; #Android; #BM_.NET maui_Android_JULY2025; #.NET MAUI SDKs; #Device; #Info and Relationship; #AINotInscope; #Talkback;

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Android
Model Name: Google Pixel 8
Android version: 15

### Repro Steps:

1. Install and open "Developer balance" app.
2. "Home Screen" will open.
3. Swipe till Hamburger menu and double tap on it, Projects options screen will open.
4. Swipe till New button and double tap on it, New project screen will appear.
5. Swipe till categories and Choose a category from the list of category and also Choose a tag from the list of tags.
6. We can select the task from the existing task list.
7. Observe that Talkback is not announcing the complete information of list items in Projects screen.

### Actual Result:
Talkback is not announcing the complete information of list items in Projects screen.
NOTE: Issue is also reproducing in IOS.

### Expected Result:
Talkback should be announcing the complete information of list items in Projects screen to users.

### User Impact:
Screen reader users will find it difficulty of knowing or accessing the complete information of those list items if, Talkback is not announcing the complete information of list items in Projects screen.

### Attachment:

Android Attachment:
https://github.com/user-attachments/assets/a9250d75-e902-42fd-bc48-eca0b3c3b637

IOS Attachment: 

[30831_Reproducing with IOS.zip](https://github.com/user-attachments/files/21467141/30831_Reproducing.with.IOS.zip)

**Comment by ghost** [comment]
This issue has been verified in Visual Studio Code 1.102.1 with MAUI versions 9.0.0 & 9.0.90. It can be reproduced on the Android platform.

**Comment by keerthiduvvuri** [comment]
Closing this bug as we are tracking only 1 bug for every platform.
Will follow up with this bug https://github.com/dotnet/maui/issues/30827
